packages/@react-stately/numberfield/src/useNumberFieldState.ts

```diff
@@ -16,6 +16,150 @@ import {NumberFieldProps} from '@react-types/numberfield';
 import {NumberFormatter, NumberParser} from '@internationalized/number';
 import {useCallback, useMemo, useState} from 'react';
 
+
+let supportedLocales = new Map<string, {groupSeparator: string, decimalSeparator: string}>([
```

Comment:
these work as group separators for `-u-nu-latn`, do they also work for `arab` and `hanidec`?

Reply:
I haven't tested that, but will work to add the additional numbering systems we support.

Reply:
it may be possible to generate all of these out of the unicode rules, like we did for plurals https://www.unicode.org/cldr/charts/44/supplemental/language_plural_rules.html#comparison
https://github.com/adobe/react-spectrum/blob/main/scripts/generateAllPlurals.mjs